service/templates/admin.html

```diff
@@ -2309,6 +2310,31 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 						</section>
 
 						<section layout="row" flex>
+							<md-card flex=50 id="cost">
+								<md-card-title>
+									<md-card-title-text>
+										<span>Cost Config</span>
+									</md-card-title-text>
+								</md-card-title>
+								<md-card-content>
+									<div>
+										Task cost calculation configuration:
+									</div>
+									<md-input-container class="control" style="width:45%;">
+										<label>Finance Formula</label>
```

Comment:
Could we maybe add explanation descriptions to the three fields? That way future on-lookers have some more context on it